documents/Libraries/stdlib/sx-osl/mx_matte_color2.inline

```diff
@@ -1 +1 @@
-vec2( {{fg}}.x*{{fg}}.w + {{bg}}.x*(1.0-{{fg}}.w), {{fg}}.w + ({{bg}}.w*(1.0-{{fg}}.w)) ) * {{mix}} + ({{bg}} * (1.0-{{mix}}));
+color2( {{fg}}.r*{{fg}}.a + {{bg}}.r*(1.0-{{fg}}.a), {{fg}}.a + ({{bg}}.a*(1.0-{{fg}}.a)) ) * {{mix}} + ({{bg}} * (1.0-{{mix}}));
```

Comment:
Not part of this review, but this direct access to sub-components of an input is what I mentioned earlier will not work for OSL in SHADER_INTERFACE_REDUCED mode when many input variables are replaced by pure values. Because in OSL it's not allowed to write color2(1,1).r. So we need to declare a local variable first.. :( It's something we need to fix in a separate change list.

Reply:
Let's close this PR but not close this task until we have reduced mode working. Don't want to accumulate things in this area. Thanks.